src/index.css

```diff
@@ -252,3 +252,128 @@ body,
 .output-panel.hidden {
   display: none;
 }
+
+.log-container {
+  font-family: monospace, monospace; /* stylelint-disable-line font-family-no-duplicate-names */
+  font-size: 85%;
+  user-select: text;
+  margin: 0 10px;
+  border: 1px solid #aaa;
+  overflow: auto;
+  white-space: pre-wrap;
+  max-height: calc(60vh - 100px);
+}
+
+.log-container [data-log-level],
```

Comment:
There's a lot of log specific styles here. Would you mind moving them into their own CSS file?

Reply:
Done.